cwf/gateway/fabfile.py

```diff
@@ -417,3 +428,6 @@ def _clean_up():
     with lcd(LTE_AGW_ROOT):
         vagrant_setup("magma_trfserver", False)
         run('pkill iperf3 > /dev/null &', pty=False, warn_only=True)
+
+class FabricException(Exception):
```

Comment:
minor nit: this appears unused. can we clean this up. Thanks

Reply:
oh yes, i forgot that. I will use the exception in my next PR anyway